add release instructions for the firmware

knmcguire opened the pull request.

krichardsson commented on release.md:
```diff
@@ -0,0 +1,53 @@
+## 4. Preparing the real release
+1. Now getting ready for the real release: set the real release tag (like '2022.03') on the crazyflie-firmware and crazyflie2-nrf firmware,
+1. Use the release actions with that tag
+1. Go to the draft release and input the generate thed release notes by  `tb ghrn bitcraze/crazyflie-firmware 2022.03` 
```
Typo

krichardsson commented on release.md:
```diff
@@ -0,0 +1,53 @@
+1. In github: Collect issues of both repositories in the “next-release” milestone. 
+1. In github: Rename the milestones to the real release, for instance “2022.03”
+1. In github actions on both firmware's repos: 
+   * select the 'release' flow. Press the dow narrow on 'run workflow'
```
typo

krichardsson commented on release.md:
```diff
@@ -0,0 +1,53 @@
+![](action_screenshot.png)
+
+## 2. Make a candidate release firmware zip folder
+1. Locally clone this repository (crazyflie-release)
```
I would suggest to remove this part. As we move the build functionality to github, I think we should stop using the toolbelt release functionality (and remove it at some point). Just build the release candidate (in crazyflie-release) and use that for testing.

krichardsson reviewed (changes requested):
Looking good! Some minor comments bellow. 

One option would be to put this in a docs directory to make it compatible with the web. Not sure there is any value in publishing it on the web as this mainly is an instruction for Bitcraze.

knmcguire commented:
Thanks kristoffer! I moved it to a docs folder now as well. 

Also I updated the doc to not use the toolbelt locally but just to generate the zipfiles directly with actions. People can just download it from the draft if they need to test it first. 

I'll merge it so that I can remove the old instructions
